quickstart-templates/msi-as-a-fic-secretless/secretless-graph-request.ps1

```diff
@@ -0,0 +1,36 @@
+#######################################################################
+<#
+Perform a 2-legged flow to acquire an access token with a managed
+identity as the credential and then use it to call Microsoft Graph.
```

Comment:
nit: maybe note that this script is for public cloud, and the `resourceUrl` and Graph endpoint needs to be configured if need to run for other clouds